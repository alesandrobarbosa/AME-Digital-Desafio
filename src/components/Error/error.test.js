import React from 'react';
import renderer from 'react-test-renderer';

import { MemoryRouter } from 'react-router-dom';
import ErrorPage from './error';

describe('#ERROR PAGE COMPONENT ==>', () => {
    it('Error page should renders without crashing', () => {
        const component = renderer.create(
            <MemoryRouter>
                <ErrorPage />
            </MemoryRouter>
        ).toJSON();
    
        expect(component).toMatchSnapshot();
    });

})

