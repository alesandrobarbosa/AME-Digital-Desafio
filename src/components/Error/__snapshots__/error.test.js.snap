// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`#ERROR PAGE COMPONENT ==> Error page should renders without crashing 1`] = `
<div>
  <span>
    Oh No!! We've got a problem!
  </span>
  <a
    href="/"
    onClick={[Function]}
  >
    Restart
  </a>
</div>
`;
